### ISSUE TITLE
protolint disable not working

### ISSUE THREAD

**Issue by nvx** [yoheimuta/protolint#86]
I added the following to my proto file on the line before an 81 character long line (not really any good way to shorten the go_package option when you want to specify the full go package path).

```
// protolint:disable MAX_LINE_LENGTH
```

And yet I still get a linter error for the next line down:

```
[test.proto:8:1] The line length is 81, but it must be shorter than 80
```

Does protolint:disable not work with MAX_LINE_LENGTH? Or have I mucked up the syntax?

**Comment by yoheimuta** [comment]
@nvx Thank you for this report.
I also noticed that protolint:disable does not completely work with MAX_LINE_LENGTH.

I'm going to work on that fix.

**Comment by yoheimuta** [comment]
I fixed it with v0.19.1. Please confirm that it works well whenever you have time.

- ref. https://github.com/yoheimuta/protolint/releases/tag/v0.19.1

**Comment by nvx** [comment]
Looks like it's working now, cheers!
